pool-interface/src/reserve_config.rs

```diff
@@ -2,55 +2,49 @@ use common::rate_math::RATE_DENOMINATOR;
 use soroban_sdk::{contracttype, Address, BytesN, Env};
 
 #[contracttype]
-pub struct ReserveConfigurationMap {
-    //bit 0-15: LTV
-    pub ltv: u32,
-    //bit 16-31: Liq. threshold
-    pub liq_threshold: u32,
-    //bit 32-47: Liq. bonus
-    pub liq_bonus: u32,
-    //bit 48-55: Decimals
+pub struct ReserveConfiguration {
     pub decimals: u32,
-    //bit 56: Reserve is active
     pub is_active: bool,
-    //bit 57: reserve is frozen
     pub is_frozen: bool,
-    //bit 58: borrowing is enabled
-    //bit 59: stable rate borrowing enabled
     pub borrowing_enabled: bool,
-    //bit 60-63: reserved
-    pub reserved: BytesN<1>,
-    //bit 64-79: reserve factor
-    pub reserve_factor: u32,
-    // A value between 0 and 100 % specifies what fraction of the underlying asset counts toward the portfolio collateral value.
+    pub liq_bonus: u32,
+    pub liq_cap: i128,
+    /// Specifies what fraction of the underlying asset counts toward
+    /// the portfolio collateral value [0%, 100%].
     pub discount: u32,
 }
 
-impl ReserveConfigurationMap {
-    fn default(env: &Env) -> Self {
+impl ReserveConfiguration {
+    fn default() -> Self {
         Self {
-            ltv: Default::default(),
-            liq_threshold: Default::default(),
             liq_bonus: Default::default(),
+            liq_cap: Default::default(),
             decimals: Default::default(),
             is_active: true,
             is_frozen: false,
             borrowing_enabled: false,
-            reserved: zero_bytes(env),
-            reserve_factor: Default::default(),
             discount: Default::default(),
         }
     }
 }
 
+#[contracttype]
+#[derive(Clone)]
+pub struct InterestRateConfiguration {
+    pub alpha: u32,
+    pub rate: u32,
+    pub max_rate: u32,
+    pub scaling_coeff: u32,
```

Comment:
```suggestion
    pub scaling: u32,
```

Reply:
Agreed on `scaling_coeff` to be consistent with the spec.